cmd/uzfs_test/zrepl_utest.c

```diff
@@ -0,0 +1,411 @@
+#include <stdio.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <strings.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+#include <netdb.h>
+#include <unistd.h>
+#include <pthread.h>
+#include <string.h>
+#include <zrepl_prot.h>
+#include <sys/zfs_context.h>
+#include <uzfs_test.h>
+#include <uzfs_mgmt.h>
+
+char *accpt_port = "3232";
+char *tgt_port = "6060";
+
+struct data_io {
+	zvol_io_hdr_t hdr;
+	char buf[0];
+};
+
+static int
+create_and_bind(char *port)
+{
+	int s, sfd;
+	struct addrinfo hints = {0, };
```

Comment:
does this initialization work?

Reply:
I think it should work.  Will check and get back to you.

Reply:
I think it should work.